code/modules/mob/living/simple_animal/abnormality/teth/Brazen_Bull.dm

```diff
@@ -0,0 +1,142 @@
+						ABNORMALITY_WORK_REPRESSION = list(50, 50, 40, 40, 40)
+						)
+	work_damage_amount = 7
+	work_damage_type = RED_DAMAGE
+	var/charge_check_time = 1 SECONDS
+	var/dash_num = 50
+	var/list/been_hit = list()
+	var/busy = FALSE
+	var/can_move = TRUE
+
+	ego_list = list(
+		/datum/ego_datum/weapon/capote,
+		/datum/ego_datum/armor/capote
+		)
+	gift_type = /datum/ego_gifts/capote
+
+	abnormality_origin = ABNORMALITY_ORIGIN_LIMBUS
+
+/mob/living/simple_animal/hostile/abnormality/brazen_bull/FailureEffect(mob/living/carbon/human/user, work_type, pe)
+	datum_reference.qliphoth_change(-1)
+
+/mob/living/simple_animal/hostile/abnormality/brazen_bull/NeutralEffect(mob/living/carbon/human/user, work_type, pe)
+	if(prob(60))
+		datum_reference.qliphoth_change(-1)
+
+/mob/living/simple_animal/hostile/abnormality/brazen_bull/Move()
+	if(!can_move)
+		return FALSE
+	return ..()
+
+/mob/living/simple_animal/hostile/abnormality/brazen_bull/Life()
+	. = ..()
+	if(!.)
+		return FALSE
+	if(!(status_flags & GODMODE))
+		if(!busy)
+			charge_check()
+
+/mob/living/simple_animal/hostile/abnormality/brazen_bull/proc/charge_check()
+	var/mob/living/carbon/human/target
+	if(busy)
+		return
+	var/list/possible_targets = list()
+	for(var/mob/living/carbon/human/H in view(20, src))
+		possible_targets += H
+	if(LAZYLEN(possible_targets))
+		target = pick(possible_targets)
+		var/dir_to_target = get_cardinal_dir(get_turf(src), get_turf(target))
+		if(dir_to_target)
+			can_move = FALSE
+			busy = TRUE
+			addtimer(CALLBACK(src, .proc/charge, dir_to_target, 0, target), 2 SECONDS)
+			return
+	return
+
+
+/mob/living/simple_animal/hostile/abnormality/brazen_bull/proc/charge(move_dir, times_ran, target)
+	setDir(move_dir)
+	var/stop_charge = FALSE
+	if(times_ran >= dash_num)
+		stop_charge = TRUE
+	var/turf/T = get_step(get_turf(src), move_dir)
+	if(!T)
+		been_hit = list()
+		stop_charge = TRUE
+		return
+	if(T.density)
+		stop_charge = TRUE
+	for(var/obj/structure/window/W in T.contents)
+		W.obj_destruction()
+	for(var/obj/machinery/door/D in T.contents)
+		if(D.density)
+			stop_charge = TRUE
+	for(var/mob/living/simple_animal/hostile/abnormality/D in T.contents)
+		if(D.density)
+			stop_charge = TRUE
+
+	if(stop_charge)
+		can_move = TRUE
+		busy = TRUE
+		addtimer(CALLBACK(src, .proc/endCharge), 9 SECONDS)
+		been_hit = list()
+		return
+	forceMove(T)
+
+	for(var/turf/U in range(1, T))
+		var/list/new_hits = HurtInTurf(U, been_hit, 0, RED_DAMAGE, hurt_mechs = TRUE) - been_hit
+		been_hit += new_hits
+		for(var/mob/living/L in new_hits)
+			L.visible_message(span_boldwarning("[src] rams [L]!"), span_userdanger("[src] impales you with its horns!"))
+			playsound(L, attack_sound, 75, 1)
+			new /obj/effect/temp_visual/kinetic_blast(get_turf(L))
+			if(ishuman(L))
+				var/mob/living/carbon/human/H = L
+				H.apply_damage(30, RED_DAMAGE, null, L.run_armor_check(null, RED_DAMAGE), spread_damage = TRUE)
+			else
+				L.adjustRedLoss(10)
```

Comment:
With this kind of damage you really *don't* need a damage difference for human and inhuman mobs. I know this was copy-pasta'd from KoG but like... 30 damage is nothing for non human mobs.